From c8e9623da211d8c268e1f034a9c1db2a58dcfbbe Mon Sep 17 00:00:00 2001
From: Greg Kroah-Hartman <gregkh@google.com>
Date: Wed, 8 Apr 2020 19:14:42 +0200
Subject: [PATCH 683/830] ANDROID: GKI: fix up gki_defconfig for x86
X-NVConfidentiality: public

CONFIG_CPU_FREQ_DEFAULT_GOV_SCHEDUTIL is now removed as it's not the
default.

Signed-off-by: Greg Kroah-Hartman <gregkh@google.com>
Change-Id: If08ceea2ca890094bb7c977b84db7ea390499319
---
 arch/x86/configs/gki_defconfig | 1 -
 1 file changed, 1 deletion(-)

diff --git a/arch/x86/configs/gki_defconfig b/arch/x86/configs/gki_defconfig
index 446484e..4cefcbb 100644
--- a/arch/x86/configs/gki_defconfig
+++ b/arch/x86/configs/gki_defconfig
@@ -53,7 +53,6 @@ CONFIG_PM_WAKELOCKS=y
 CONFIG_PM_WAKELOCKS_LIMIT=0
 # CONFIG_PM_WAKELOCKS_GC is not set
 CONFIG_CPU_FREQ_TIMES=y
-CONFIG_CPU_FREQ_DEFAULT_GOV_SCHEDUTIL=y
 CONFIG_CPU_FREQ_GOV_POWERSAVE=y
 CONFIG_CPU_FREQ_GOV_CONSERVATIVE=y
 CONFIG_IA32_EMULATION=y
-- 
2.7.4

